airdrop/constants.py

```diff
@@ -24,10 +24,22 @@
     },
     "required": ["signature", "address", "airdrop_id", "airdrop_window_id", "block_number"],
 }
+OCCAM_CLAIM_SCHEMA = {
```

Comment:
OCCAM_CLAIM_SCHEMA and CLAIM_SCHEMA are the same? Do we need different structures?

Reply:
Nope we don't need different structure. Will reuse the one defined structure.